lib/service_base_url_test_kit/service_base_url_validate_group.rb

```diff
@@ -324,33 +450,66 @@ def skip_message
         end
 
         organization_resources.each do |organization|
-          assert !organization.address.empty?,
-                 "Organization with id: #{organization.id} does not have the address field populated"
+          resource_is_valid?(resource: organization)
+
+          if organization.address.empty?
+            add_message('error', "Organization with id: #{organization.id} does not have the address field populated")
+          end
 
           if organization.endpoint.empty?
-            assert organization.partOf.present?, %(
-                    Organization with id: #{organization.id} does not have the endpoint or partOf field populated
-                  )
+            if organization.partOf.blank?
+              add_message('error', %(
+                Organization with id: #{organization.id} does not have the endpoint or partOf field populated
+              ))
+              next
+            end
+
             parent_organization = find_parent_organization(bundle_resource, organization.partOf.reference)
-            assert(parent_organization.present?, %(
-              Organization with id: #{organization.id} references parent Organization not found in the Bundle:
-              #{organization.partOf.reference}
-            ))
-            assert(!parent_organization.endpoint.empty?, %(
-              Organization with id: #{organization.id} has parent Organization with id: #{parent_organization.id} that
-              does not have the `endpoint` field populated.
-            ))
+
+            if parent_organization.blank?
+              add_message('error', %(
+                Organization with id: #{organization.id} references parent Organization not found in the Bundle:
+                #{organization.partOf.reference}
+              ))
+              next
+            end
+
+            if parent_organization.endpoint.empty?
+              add_message('error', %(
+                Organization with id: #{organization.id} has parent Organization with id: #{parent_organization.id} that
+                does not have the `endpoint` field populated.
+              ))
+            end
           else
             endpoint_references = organization.endpoint.map(&:reference)
 
             endpoint_references.each do |endpoint_id_ref|
               organization_referenced_endpts = find_referenced_endpoint(bundle_resource, endpoint_id_ref)
-              assert !organization_referenced_endpts.empty?,
-                     "Organization with id: #{organization.id} references an Endpoint that is not contained in this
-                    bundle."
+              next unless organization_referenced_endpts.empty?
+
+              add_message('error', %(
+                Organization with id: #{organization.id} references an Endpoint that is not contained in this
+                bundle."
```

Comment:
```suggestion
                bundle.
```

Don't need trailing quote.